DEVX-1041: Fix ksql websocket initializor in rbac-docker

jonchiu opened the pull request:
rbac-docker example was using incorrect security handler. The fix was to set it to the same handler used by the rest servlet: InstallBearerOrBasicSecurityHandler

ybyzek reviewed (changes requested):
@jonchiu LGTM.  However, can we rebase/merge into 5.3.0-post instead of 5.3.x?

jonchiu committed:
DEVX-1041: fix ksql websocket servlet initializor classes

rbac-docker example was using incorrect security handler. The
fix was to set it to the same handler used by the rest servlet:
InstallBearerOrBasicSecurityHandler

jonchiu commented:
@ybyzek ping
